core/ledger/util/couchdb/couchdbutil.go

```diff
@@ -48,8 +49,20 @@ func CreateCouchInstance(couchDBConnectURL, id, pw string, maxRetries,
 	// and for efficiency should only be created once and re-used.
 	client := &http.Client{Timeout: couchConf.RequestTimeout}
 
-	transport := &http.Transport{Proxy: http.ProxyFromEnvironment}
-	transport.DisableCompression = false
+	transport := &http.Transport{
```

Comment:
As, it is merged in master, I am fine with this backport. However, just to take an additional note, I would have preferred this change the following way...
``` transport := http.DefaultTransport.(*http.Transport).Clone()
      transport.MaxIdleConns = 2000
      transport.MaxIdleConnsPerHost = 2000
```
As, this makes it easy to read what is changed in the defaults and takes care if default values are changed or new fields are added in the future versions. However, I did not like the casting part in this. Nonetheless, it appears that the "Clone" function on "http.Transport" was added specifically to solve this problem (https://github.com/golang/go/issues/26013).

Reply:
Agreed, @manish-sethi . Since the beginning, the master didn't use the clone of `DefaultTransport`.  https://github.com/hyperledger/fabric/blob/22a49625a7e844d908fec2ff56ddb3f93f3c616c/core/ledger/util/couchdb/couchdbutil.go#L57

When I made the changes in master, I appended the other parameters to the existing definition. I wasn't aware of this clone operation (thanks for pointing this out). 
https://github.com/hyperledger/fabric/blob/28c6efdaaffd1d7e17859b0ae98e0bb9594b3eaa/core/ledger/util/couchdb/couchdbutil.go#L65

On a different note, `http.DefaultTransport` https://github.com/golang/go/blob/master/src/net/http/transport.go#L42 is a package level state right? Hence, I could defend by saying that we are against defining package-level states and also the usage of existing package-level states (on a funny note).